cmd/packageloader/loader/loader.go

```diff
@@ -0,0 +1,233 @@
+package loader
+
+import (
+	"bufio"
+	"errors"
+	"io/fs"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/skycoin/cx/cmd/packageloader/bolt"
+	"github.com/skycoin/cx/cmd/packageloader/redis"
+	"golang.org/x/crypto/blake2b"
+)
+
+var DATABASE = "bolt"
+var SRC_PATH string
+var CURRENT_PATH string
+var IMPORTED_DIRECTORIES = []string{}
```

Comment:
Also, instead of using globals for these, just pass these as arguments to the functions that need these vars